components/net/Cargo.toml

```diff
@@ -7,11 +7,12 @@ workspace = "../../"
 [dependencies]
 bitflags = "*"
 fnv = "*"
+habitat_builder_protocol = { path = "../builder-protocol" }
+habitat_core = { path = "../core" }
```

Comment:
Yeah! Was wondering if anyone saw that I did this with `components/sup/Cargo.toml` as a test